src/agent.ts

```diff
@@ -54,6 +56,33 @@ export class Agent {
     this.connection = connection;
   }
 
+  /**
+   * List all agents in the current project.
+   */
+  public static async list(project: SfProject): Promise<string[]> {
```

Comment:
needs to be static b/c we don't have a `Connection` instance to pass to the constructor in vscode